creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -112,6 +113,7 @@ const syncRouteController = async (req, res) => {
 
   if (immediate) {
     try {
+      tracing.info('processing manual immediate sync')
```

Comment:
do you have an example of what this looks like on the dashboard?

Reply:
<img width="790" alt="Screen Shot 2022-08-30 at 08 28 57" src="https://user-images.githubusercontent.com/17736112/187436704-9394addd-1f5e-4c76-90c6-922dbdf71a9f.png">

Like this